File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/OrderDailyEntity.java
package com.robinfood.core.entities;

import com.robinfood.core.entities.services.OriginEntity;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class OrderDailyEntity {

    private final Long brandId;

    private final String createdAt;

    private final Long deliveryTypeId;

    private final Long id;

    private final String orderInvoiceNumber;

    private final String orderNumber;

    private final OriginEntity origin;

    private final Double total;

    private final OrderDetailUserEntity user;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/orderdaily/OrderDailyEntity.kt
package com.robinfood.core.entities.orderdaily

import java.math.BigDecimal

data class OrderDailyEntity(
    val brand: OrderBrandDailyEntity,
    val brandId: Long,
    val createdAt: String,
    val deliveryTypeId: Int,
    val id: Long,
    val orderInvoiceNumber: String,
    val orderNumber: String,
    val origin: OrderOriginDailyEntity,
    val total: BigDecimal,
    val user: OrderDailyUserEntity
)
